apps/api-extractor/src/analyzer/AstSymbolTable.ts

```diff
@@ -428,11 +432,23 @@ export class AstSymbolTable {
         // This handles cases such as "export default class X { }" where the symbol name is "default"
         // but the declaration name is "X".
         localName = followedSymbol.name;
-        for (const declaration of followedSymbol.declarations || []) {
-          const declarationNameIdentifier: ts.DeclarationName | undefined = ts.getNameOfDeclaration(declaration);
-          if (declarationNameIdentifier && ts.isIdentifier(declarationNameIdentifier)) {
-            localName = declarationNameIdentifier.getText().trim();
-            break;
+        if (TypeScriptHelpers.isWellKnownSymbolName(localName)) {
+          localName = `[Symbol.${localName.slice(3)}]`;
```

Comment:
Could you add a code comment showing an example of the `localName` value before and after this transformation?  i.e. what are the 3 characters that we're removing?